### PR TITLE
✨Source Yahoo-Finance-Price: Migrate Python CDK to No-code CDK 

### LINKED ISSUE #28920: yahoo-finance-price

### PR DESCRIPTION
**Pull request by masyagin1998**
## What
* add no-code version of Yahoo-Finance-Price connector (closes #28920)

## How
* created Yahoo-Finance-Price no-code version in Airbyte Connector Builder and then put it to new source

## Recommended reading order
* manifest.yaml

## 🚨 User Impact 🚨
* no

## Pre-merge Actions
* review

### CODE REVIEW

**Comment by masyagin1998** [comment]
@marcosmarxm @sajarin ready for review!

**Comment by masyagin1998** [comment]
Screenshot from 2023-08-12 11-16-46
Screenshot from 2023-08-12 11-17-13

After upgrading Airbyte there is one error in acceptance test connected with `dagger`. Except this error everything is fine.

**Comment by marcosmarxm** [comment]
Hello 👋 , right now Airbyte's integration tests are failing for this connector and I'm going to work during the week to fix them. Due a high number of contributions we have a backlog and your contribution will be review next week. We ask to you share the integration test from your side whatever possible and if you don't have credentials to run tests to get in touch with @RealChrisSean as soon as possible. Also some common missing steps contributors does migrating to low code are (and you can double check to make sure you implemented this): 
* adding paginator
* update the documentation changelog + metadata file and dockerfile version
* if the connector has `start_date` parameter you must implement incremental syncs

File: airbyte-integrations/connectors/source-yahoo-finance-price/Dockerfile [hunk 1]
```diff
@@ -34,5 +34,5 @@ COPY source_yahoo_finance_price ./source_yahoo_finance_price
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.3
+LABEL io.airbyte.version=0.1.0
```
[the remark this answers is not in the document]
Comment (masyagin1998) [reply airbyte-integrations/connectors/source-yahoo-finance-price/Dockerfile:37]: Okay, thank you!!

**Review by marcosmarxm — APPROVED** [review]
Thanks @masyagin1998

**Pull request merged**